packages/video_player/lib/video_player.dart

```diff
@@ -32,13 +32,34 @@ class DurationRange {
 }
 
 class VideoPlayerValue {
+  /// The total duration of the video.
+  ///
+  /// Before the [VideoPlayerController] is initialized this is null.
   final Duration duration;
+
+  /// The current playback position.
   final Duration position;
+
+  /// The currently buffered ranges.
   final List<DurationRange> buffered;
+
+  /// True if the video is playing. False if it's paused.
   final bool isPlaying;
+
+  /// True iff the video is looping.
   final bool isLooping;
+
+  /// The current volume of the playback.
   final double volume;
+
+  /// A description of the error.
+  ///
+  /// If [isErroneous] is false this is [null].
```

Comment:
Could we change `isErroneous` to `hasError` as in the camera plugin?
(Would be a breaking change -> version 0.2.0, I guess)

Reply:
Done